stl/src/vector_algorithms.cpp

```diff
@@ -1136,7 +1136,7 @@ _Min_max_element_t __stdcall __std_minmax_element_8(
 
 namespace {
     template <class _Ty>
-    const void* _Find_trivial_unsized_fallback(const void* _First, _Ty _Val) {
+    const void* _Find_trivial_unsized_fallback(const void* _First, _Ty _Val) noexcept {

```

Comment:
#4113 is ready to merge and is removing these 4 functions (by manually inlining them). I'll push a change to pre-emptively resolve this merge conflict by dropping the `noexcept`s here. (The `_Minmax_traits_8::_Sign_correction` change above is still righteous.)